packages/lesswrong/components/common/AnalyticsTracker.jsx

```diff
@@ -0,0 +1,24 @@
+import { registerComponent } from 'meteor/vulcan:core';
+import React, { useCallback } from 'react';
+import { useTracking } from "../../lib/analyticsEvents";
+
+const buttonDecoding = {
+  0: 'main_button',
+  1: 'auxilliary_button',
+  2: 'secondary_button',
+}
+const AnalyticsTracker = ({eventType, eventProps, children, captureOnClick, captureOnMount, skip}) => {
```

Comment:
should be:

captureOnClick=true

and then, remove all instances of captureOnClick